src/lib/modernExtend.ts

```diff
@@ -1611,21 +1628,31 @@ export function iasWarning(args?: IasWarningArgs): ModernExtend {
 type MultiplierDivisor = {multiplier?: number; divisor?: number};
 export interface ElectricityMeterArgs {
     cluster?: 'both' | 'metering' | 'electrical';
+    electricalMeasurementType?: 'both' | 'ac' | 'dc';
     current?: false | MultiplierDivisor;
     power?: false | MultiplierDivisor;
     voltage?: false | MultiplierDivisor;
     energy?: false | MultiplierDivisor;
     producedEnergy?: false | true | MultiplierDivisor;
     acFrequency?: false | true | MultiplierDivisor;
     threePhase?: boolean;
-    configureReporting?: boolean;
+    configureReporting?: false | true | ReportingConfigWithoutAttribute;
```

Comment:
I would keep this `false`/`true` and allow to pass different min/max/change per attribute (current/power/etc. like done in https://github.com/Koenkk/zigbee-herdsman-converters/pull/8123/files)